src/main/java/org.wso2.apim.monetization/impl/model/GraphQLClient.java

```diff
@@ -0,0 +1,11 @@
+package org.wso2.apim.monetization.impl.model;
```

Comment:
missing license header

Reply:
Fixed in 09f0f2a0ec6b148f13a8092bf53055c9df346ef2